Cargo.toml

```
@@ -32,6 +32,7 @@ rust-version = "1.76.0"
 [workspace.dependencies]
 log = { version = "0.4", default-features = false }
 qlog = { version = "0.13", default-features = false }
+quinn-udp = { version = "0.5.4", default-features = false, features = ["direct-log"] }
```

```suggestion
quinn-udp = { version = "0.5", default-features = false, features = ["direct-log"] }
```
to be consistent with how we not specify minor versions for other deps? Or do we need this here?

---

Sorry for not specifically calling this out. We need `v0.5.4` or above for the following reasons:

- `direct-log` feature is only available with `v0.5.3`, see https://github.com/quinn-rs/quinn/pull/1923
- `v0.5.3` had a bug, fixed in `v0.5.4` https://github.com/quinn-rs/quinn/pull/1932

The above, i.e. `v0.5.4` allows cargo to choose any version `>=0.5.4`.